chatbot_app/main.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import FastAPI
```

Comment:
load_dotenv w sumie moze byc w punkcie startowym aplikacji np w startup.py nie musi byc w module alternatywnie .env ladowany przez liba ktory zarzadza konfiguracja.
zmienna CONNECTION_STRING - wczytujesz ja ale nie uzywasz wiec nie wiem po co jest :D 

Mozesz dodac routery i strukture projektu np api/routers i wwtedy bedzie ci latwiej skalowac to pozniej

Przydaloby sie typowac endpointy np : 
@app.get("/", response_model=dict)
def read_root() -> dict:
    return {"message": "Welcome"}